model_zoo/app_integrate_bundle/ensemble.py

```diff
@@ -179,17 +170,22 @@ def __call__(self, args, datalist, test_datalist=None):
             device = self._device(args.device)
             logger.info(f"Using device: {device}")
 
-            overrides = {
-                Const.KEY_BUNDLE_ROOT: self.bundle_path,
-                Const.KEY_TRAIN_TRAINER_MAX_EPOCHS: max_epochs,
-                Const.KEY_TRAIN_DATASET_DATA: _train_ds,
-                Const.KEY_VALIDATE_DATASET_DATA: _val_ds,
-                Const.KEY_DATASET_DIR: dataset_dir,
-                Const.KEY_MODEL_PYTORCH: model_pytorch,
-                Const.KEY_DEVICE: device,
-            }
+            self.train_workflow.bundle_root = self.bundle_path
+            self.train_workflow.max_epochs = max_epochs
+            self.train_workflow.train_dataset_data = _train_ds
+            self.train_workflow.val_dataset_data = _val_ds
+            self.train_workflow.dataset_dir = dataset_dir
+            self.train_workflow.device = device
+            self.train_workflow.initialize()
```

Comment:
I think these `setting` operations are only useful for single GPU training, so we should move them to line 229, right?

Thanks.

Reply:
Yes, have removed them, thanks!